infra/dogfood/templates/livekit.yaml

```
@@ -0,0 +1,8 @@
+port: 17880
+rtc:
+  tcp_port: 17881
+  udp_port: 17882
+  use_external_ip: false
+keys:
+  replace-key: replace-secret
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)